dkroutingtool/src/py/manual_viz.py

```diff
@@ -41,6 +43,15 @@ def write_manual_output(node_data, routes_for_mapping, vehicles, zone_route_map)
                 else:
                     node_num.append(node_i)
                     node_i += 1
+
+            #formulas
+            node_name.insert(formula_index,f'=COUNTIF(A:A,"{route_id}")-2')
```

Comment:
Is this meant to be in this pull request too? seems like a duplicate from the other

Reply:
That's because I branched off this one to start the other PR